eng/helix/content/runtests.sh

```diff
@@ -76,8 +76,17 @@ sync
 exit_code=0
 
 echo "Restore: dotnet restore RunTests/RunTests.csproj --ignore-failed-sources"
+
+# --verbosity diagnostic can be removed when random failures are identified
```

Comment:
Maybe file an issue tracking this removal

Reply:
Done in internal